modules/cluster-node-tuning-operator-verify-profiles.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id="verifying-tuned-profiles-are-applied_{context}"]
+=  Verifying that the tuned profiles are applied
+
+Use this procedure to check which tuned profiles are applied on every node.
+
+.Procedure
+
+. Check which tuned Pods are running on each node:
++
+----
+$ oc get pods -n openshift-cluster-node-tuning-operator -o wide
+----
++
+----
```

Comment:
Suggest adding `.Example output` on the output code blocks throughout.

Reply:
Added the one I missed!